db/seeds.rb

```diff
@@ -5,3 +5,10 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+Greeting.create(message:"Hello, welcome to our website!")
```

Comment:
**Kudos for adding seed!**

Reply:
@Rustamxon7 Thanks for highlighting this. I will fix them.